### PR TITLE
- fix: removed circular dependencies

### PR DESCRIPTION
**Pull request by alexnaiman**
## :loudspeaker: Type of change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description
I checked for circular dependencies using `madge . --extensions ts,tsx -c --warning` and fixed the issue (moved one interface to its own module)

## :bulb: Motivation and Context

This PR started as a fix for #479, but that issue isn't available anymore, but since I found other cyclic dependencies I removed those

## :green_heart: How did you test it?
I didn't introduce any major changes, just remade some imports. I just run all the existent tests

## :pencil: Checklist

- [ ] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] All tests passing
- [x] No breaking changes

## :crystal_ball: Next steps
I am looking forward to someone to merge this and maybe add the `hacktoberfest-accepted` label to it 😄

### CODE REVIEW

File: src/js/backend.ts [hunk 1]
```diff
@@ -5,47 +5,10 @@ import { Event, EventHint, Severity, Transport } from "@sentry/types";
 // @ts-ignore LogBox introduced in RN 0.63
 import { Alert, LogBox, YellowBox } from "react-native";
 
+import { ReactNativeOptions } from './reactNativeOptions';
```
Comment (bruno-garcia) [inline src/js/backend.ts:8]: Could you please name the file `options.ts` to keep it consistent with the others?

```suggestion
import { ReactNativeOptions } from './options';
```

**Review by bruno-garcia — CHANGES_REQUESTED** [review]
Nice! Thanks for improving this.
I left 1 note.

**Comment by bruno-garcia** [comment]
Label added :)

File: src/js/backend.ts (hunk 1 above)
Comment (alexnaiman) [reply src/js/backend.ts]: done :)

**Comment by alexnaiman** [comment]
Done! Thanks a lot for the label :)

**Review by jennmueng — APPROVED** [review]
This looks good to me.

_FYI the e2e tests will fail as non-Sentry-member PRs don't have access to Github Secrets, don't worry about that._

**Review by bruno-garcia — APPROVED** [review]
LGTM! THanks!

**Pull request merged**